content/14.biological-interpretation.md

```diff
@@ -1,8 +1,132 @@
+
+A network is a representation of the relations between objects. 
+Nodes are the entities of the network (users of a social platform, train stations, proteins), while edges are the connections between them (friendship, routes, and protein interactions, respectively). 
+In the case of protein-protein interactions, the evidence for the functional associations between proteins can be obtained experimentally. 
+For example, co-immunoprecipitation, crosslinking, and proximity labeling can be used to reveal physical interactions [@PMID:34046695]. 
+The data is presented in a table with nodes and edges (e.g., "protein A interacts with protein B") from which the network can be constructed. 
+A considerable wealth of protein-protein association data is stored in free databases like IntAct, which contain interactions derived from literature curation or direct user submissions [@PMID:34761267]. 
+Protein interactions can also be predicted by classifiers that consider many features, like orthology and co-localization, to produce a posterior odds ratio of interaction [@PMID:21572181; @PMID:33237311]. 
+Finally, large repositories like STRING (Search Tool for the Retrieval of Interacting Genes/Proteins) collect and integrate protein-protein interaction data from several databases [@PMID:33237311]. 
+It also provides a web-based interface to survey the data, and users only have to feed a search box with the identifiers of the protein(s) of interest. 
+STRING will retrieve the network and show the evidence supporting each interaction. 
+Importantly, these databases do not indicate the direction of the interaction, so they produce undirected networks. 
+If edges have directions (e.g., A influences B and not vice versa), then the network is directed. Signaling pathways are examples of directed graphs. 
+
+### Network analysis
+
+Network analysis is a group of techniques that explore and investigate the network, yielding valuable knowledge about its structure and unveiling key players regulating the flow of information. 
+One of the first steps in network analysis relates to centrality measurements. 
+Centralities are indicators of the relative importance of a node corresponding to its position in the network, and each centrality measure provides new insights to interpret the data in new ways [@PMID:27616995; @PMID:30064421].
+
+#### Degree centrality
+  
+The degree of a node measures the number of edges incident to that node. 
+Nodes with a high degree interact with many other nodes, called first neighbors. 
+In particular, the node degree distribution in protein networks is highly skewed, with most nodes having a low degree and a few having high degrees, known as hubs.
+Hubs are usually regulatory proteins, being notable examples oncogenes and transcription factors. 
+Moreover, hubs are attractive targets for directed interventions, as their alteration has a profound effect on the stability of the network [@PMID:16751849]. 
+
+#### Closeness centrality
```

Comment:
can you add a citation to the text below about closeness centrality where people can look for more information?

Reply:
Closeness centrality is not as heavily used as betweenness in protein networks. Still, I could add a nice citation showing that essential genes in yeast have a higher closeness than non-essential ones. I also added a link to a great resource at the European Bioinformatics Institute, but I don´t know if citing web pages is appropriate for this manuscript.